Patch-Source: https://gitlab.gnome.org/TheEvilSkeleton/Refine/-/merge_requests/27
---
From 673b3368aa5b572ceebd79f3b7f4c34a8ea1a29b Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Sun, 2 Feb 2025 00:10:03 +0100
Subject: [PATCH] build: Relax blueprint-compiler version

blueprint-compiler is already at 0.16.0 at works without problems, while
Refine pins the version to 0.14.0
---
 refine/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/refine/meson.build b/refine/meson.build
index acf5970..ed1b72e 100644
--- a/refine/meson.build
+++ b/refine/meson.build
@@ -23,7 +23,7 @@ blueprints = custom_target('blueprints',
   output: '.',
   command: [
     find_program('blueprint-compiler',
-      version: '0.14.0',
+      version: '>=0.14.0',
       version_argument: '--version',
     ), 'batch-compile', '@OUTPUT@', '@CURRENT_SOURCE_DIR@', '@INPUT@'
   ],
-- 
GitLab

